pkg/constant/const.go

```diff
@@ -118,6 +118,10 @@ const (
 	OpsRequestNameLabelKey                   = "ops.kubeblocks.io/ops-name"
 	ServiceDescriptorNameLabelKey            = "servicedescriptor.kubeblocks.io/name"
 	RestoreForHScaleLabelKey                 = "apps.kubeblocks.io/restore-for-hscale"
+	ResourceConstraintProviderLabelKey       = "resourceconstraint.kubeblocks.io/provider"
+
+	// StatefulSetPodNameLabelKey is used to mark the pod name of the StatefulSet
+	StatefulSetPodNameLabelKey = "statefulset.kubernetes.io/pod-name"
```

Comment:
statefulset will be removed in 0.9.0

Reply:
> statefulset will be removed in 0.9.0

Yes, if we're certain to remove sts in 0.9, we can refactor and make changes at that time. I've noted the relevant TODOs.